gateway-cm-integration/src/main/java/org/apache/knox/gateway/SupportedService.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.knox.gateway;
+
+import static java.util.Collections.emptySet;
+import static java.util.Collections.singleton;
+
+import java.util.Set;
+
+public enum SupportedService {
```

Comment:
So I don't quite understand the purpose of this. I left a comment on the Jira about how dependencies between services isn't specific to CM. I don't know why certain services would be listed here. This seems brittle to have to keep up to date.